README.md

```diff
@@ -1,15 +1,17 @@
 # Community Committee
-The Community Committee reflects a formal role and the relevance of the voice of community for the governance of Node.js Foundation.
+The Community Committee reflects a formal voice for the community in the Node.js Foundation.
 
-Community activities, such as ecosystem projects and events, fall outside of the realm of code and documentation that the TSC oversees. Node.js' leadership is instantiated in a Board of Directors. There are are number of organizations that have absolutely contributed to the growth of Node.js--NodeSchool, NodeBots, and thousands of individual meetups. This is not an exhaustive list.The champions of these organizations have very different skillsets, talents, and concerns. The community organizations of Node.js have long been important to the growth and vitality of the project and will be recognized as the Community Committee(CC) as represented on the Board by the existing Individual Membership Board Directors.
+Community activities, such as ecosystem projects and events, fall outside of core projects that the TSC oversees. The Node.js Foundation's is governed by its Board of Directors. There are are number of outside organizations that have absolutely contributed to the growth of Node.js--NodeSchool, NodeBots, and thousands of individual meetups. This is not an exhaustive list. The champions of these organizations have very different skillsets, talents, and concerns. These community organizations of Node.js have long been important to the growth and vitality of the Node.js ecosystem and should be respresented by the Community Committee(CC) which in turn can represent their interested and aquire support from the Node.js Foundation through its Board of Directors.
```

Comment:
`Foundation's is governed` -> `Foundation is governed`

(This might be easier to review and comment on if it were wrapped at 80 chars or something.)